src/components/LeafletMap.js

```diff
@@ -269,8 +276,89 @@ export class LeafletMap {
           </div>
       `;
       marker.bindPopup(markerContent).openPopup();
+      this.showSearchResultProtections(resource);
     } catch (e) {
       dispatch.call("search-bar-no-data", this, e);
     }
   };
+
+  showSearchResultProtections = async (resource) => {
+    let protections = await this.getSearchResultProtections(resource);
+    console.log("protections:");
+    protections.forEach((key, val) => {
+      console.log(`${key}:${val}`);
+    });
+    dispatch.call("render-infowindow", null, {
+      template: "searchResult",
+      data: protections,
+    });
+  };
+
+  getSearchResultProtections = async (resource) => {
+    //this is necessary because Bing sometimes returns full state names and other times as two-letter abbreviations
```

Comment:
Wonder if bing has a way to set this?

Reply:
I don't think it does; I was searching through the docs. There is no consistency; it will return the abbreviation for some places and name for others.